doc/admin-api.md

```diff
@@ -513,7 +519,9 @@ In addition to the basic complex equalization algorithm selection, APISIX's Upst
 |desc     |optional|upstream usage scenarios, and more.|
 |pass_host            |optional|`pass` pass the client request host, `node` not pass the client request host, using the upstream node host, `rewrite` rewrite host by the configured `upstream_host`.|
 |upstream_host    |optional|This option is only valid if the `pass_host` is `rewrite`.|
-|labels|False |Match Rules|Key/value pairs to specify attributes|{"version":"v2","build":"16","env":"production"}|
+|labels|optional |Key/value pairs to specify attributes|{"version":"v2","build":"16","env":"production"}|
+|create_time|optional| epoch timestamp in second, like `1602883670`|
+|update_time|optional| epoch timestamp in second, like `1602883670`|
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/98954206-841fb980-2538-11eb-87f8-6cf0d78f7e8f.png)

We may format all docs to have a uniform style in the future.